plugins/statistics.koplugin/main.lua

```diff
@@ -0,0 +1,424 @@
+local InputContainer = require("ui/widget/container/inputcontainer")
+local MultiInputDialog = require("ui/widget/multiinputdialog")
+local CenterContainer = require("ui/widget/container/centercontainer")
+local UIManager = require("ui/uimanager")
+local Screen = require("device").screen
+local DEBUG = require("dbg")
+local Menu = require("ui/widget/menu")
+local Font = require("ui/font")
+local _ = require("gettext")
+local TimeVal = require("ui/timeval")
+local dump = require("dump")
+local lfs = require("libs/libkoreader-lfs")
+local tableutil = require("tableutil")
+local T = require("ffi/util").template
```

Comment:
This is total nitpicking, but I do believe there is a touch of organization to the requires, with the main stuff generally ending with

```lua
local DEBUG = require("dbg")
local T = require("ffi/util").template
local _ = require("gettext")
```

But perhaps I'm imagining that. If I am I would still like it to be the case. :P